openmetadata-ui/src/main/resources/ui/src/axiosAPIs/miscAPI.ts

```diff
@@ -75,7 +75,7 @@ export const getSuggestions: Function = (
   return APIClient.get(
     `/search/suggest?q=${queryString}&index=${
       searchIndex ??
-      `${SearchIndex.DASHBOARD},${SearchIndex.TABLE},${SearchIndex.TOPIC},${SearchIndex.PIPELINE}`
+      `${SearchIndex.DASHBOARD},${SearchIndex.TABLE},${SearchIndex.TOPIC},${SearchIndex.PIPELINE},${SearchIndex.MLMODEL}`
```

Comment:
can we please use queryParams as an object and not append it manually ?

Reply:
@yug-shah0106 , Yes, will update this.